core/station/api/spec.did

```diff
@@ -2770,24 +2784,136 @@ type InitAssetInput = record {
   metadata : vec AssetMetadata;
 };
 
-// The init configuration for the canister.
+// The input type for creating a user group when initializing the canister for the first time.
+type InitUserGroupInput = record {
+  // The id of the user group.
+  id : opt UUID;
+  // The name of the user group, must be unique.
+  name : text;
+};
+
+// The input type for adding identities to a user.
+type UserIdentityInput = record {
+  // The identity of the user.
+  identity : principal;
+};
+
+// The users to create when initializing the canister for the first time.
+type InitUserInput = record {
+  // The id of the user, if not provided a new UUID will be generated.
+  id : opt UUID;
+  // The name of the user.
+  name : text;
+  // The identities of the user.
+  identities : vec UserIdentityInput;
+  // The user groups to associate with the user (optional).
+  // If not provided it defaults to the `admin` group if default user groups are created,
```

Comment:
```suggestion
  // If not provided it defaults to the `Admin` group if default user groups are created,
```
as the doc comment of `InitialConfig` defines the admin group as `Admin` (upper-case)

Reply:
Updating